commands/update.go

```diff
@@ -0,0 +1,193 @@
+		authToken = table.GithubAccessToken
+		updatedRepos = table.UpdatedRepositories
+		transferredRepos = table.TransferredRepositories
+	}); err != nil {
+		return err
+	}
+
+	for _, repo := range transferredRepos {
+		from := repoOwner(repo.OriginalURL) + "/" + repoName(repo.OriginalURL)
+		to := repo.TargetOrg + "/" + repoName(repo.OriginalURL)
+		newURL := strings.Replace(repo.OriginalURL, from, to, -1)
+
+		if sliceutil.IsStringInSlice(newURL, updatedRepos) {
+			fmt.Println("Skip (already updated):", newURL)
+			continue
+		}
+
+		tempDir, err := pathutil.NormalizedOSTempDirPath("repo")
+		if err != nil {
+			log.Errorf("%s", err)
+			continue
+		}
+
+		if out, err := command.New("git", "clone", newURL, tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if out, err := command.New("git", "branch", "git-url-update").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if out, err := command.New("git", "checkout", "git-url-update").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if err := filepath.Walk(tempDir, func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+
+			if info.IsDir() && info.Name() == ".git" {
+				return filepath.SkipDir
+			}
+
+			if info.IsDir() {
+				return nil
+			}
+
+			c, err := ioutil.ReadFile(path)
+			if err != nil {
+				return err
+			}
+
+			c = []byte(strings.Replace(string(c), from, to, -1))
+
+			return ioutil.WriteFile(path, c, info.Mode())
+		}); err != nil {
+			log.Errorf("%s", err)
+			continue
+		}
+
+		if out, err := command.New("git", "add", ".").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if out, err := command.New("git", "commit", "-m", "updated git urls").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if out, err := command.New("git", "push", "origin", "git-url-update").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		// opening PR
+		var b bytes.Buffer
+		if err := json.NewEncoder(&b).Encode(map[string]interface{}{
+			"title": "Update repository URLs",
+			"head":  "git-url-update",
+			"base":  "master",
+		}); err != nil {
+			log.Errorf("%s", err)
+			continue
+		}
+
+		url := fmt.Sprintf("https://api.github.com/repos/%s/%s/pulls", repoOwner(newURL), repoName(newURL))
+
+		req, err := http.NewRequest(http.MethodPost, url, &b)
+		if err != nil {
+			log.Errorf("%s", err)
+			continue
+		}
+
+		req.Header.Add("Authorization", "token "+authToken)
+
+		if dumped, err := httputil.DumpRequest(req, true); err == nil {
```

Comment:
Same as previously: forgot to handle if `httputil.DumpRequest` returns error.

Reply:
same as previously